build/grunt.js

```diff
@@ -1,86 +1,26 @@
-import {gruntCustomizer, gruntOptionsMaker} from './options-customizer.js';
```

Comment:
I don't think this file is used anywhere, so, it could be removed.

Reply:
I think the file in question is already gone.

Reply:
ah, missed that because it's a collapsed diff